device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala

```diff
@@ -86,6 +86,9 @@ class GroupsResourceSpec extends FunSuite with ResourceSpec {
         }
       }
     }
+
+    deleteDeviceOk(device1)
+    deleteDeviceOk(device2)
```

Comment:
I now this is necessary but is quite a code smell if you need to cleanup like this so that other tests pass.